spec/services/order_cycle_form_spec.rb

```diff
@@ -223,22 +223,93 @@
 
     context "updating payment methods" do
       context "and it's valid" do
-        it "saves the changes" do
-          distributor = create(:distributor_enterprise)
-          distributor_payment_method = create(
-            :payment_method,
-            distributors: [distributor]
-          ).distributor_payment_methods.first
-          order_cycle = create(:distributor_order_cycle, distributors: [distributor])
+        let!(:distributor){ create(:distributor_enterprise) }
+        let!(:payment_method){ create(:payment_method, distributors: [distributor]) }
+        let!(:payment_method2){ create(:payment_method, distributors: [distributor]) }
+        let!(:distributor_payment_method){ payment_method.distributor_payment_methods.first }
+        let!(:distributor_payment_method2){ payment_method2.distributor_payment_methods.first }
```

Comment:
It looks like these variables point to the same payment methods in the two lines above, and seem unnecessary. Can't the specs refer to `payment_method` and `payment_method2`?

Reply:
oh, yes that's correct they're pointing to the same payment method.